[BUG][BYT] Build failure when CONFIG_GDB_DEBUG=y

**Describe the bug**
Enabling the GDB stub for BYT currently ends up in a build error.

**To Reproduce**
    make menuconfig
    Select Debug --> GDB Stub
    make

    sof/src/debug/gdb/ringbuffer.c:12:44: error: 'SRAM_DEBUG_BASE' undeclared here (not in a function); did you mean 'MAILBOX_DEBUG_BASE'?
    volatile struct ring * const rx = (void *) SRAM_DEBUG_BASE;
                                                ^~~~~~~~~~~~~~~
                                                MAILBOX_DEBUG_BASE

--- a/src/debug/gdb/ringbuffer.c
+++ b/src/debug/gdb/ringbuffer.c
@@ -5,13 +5,14 @@
 // Author: Marcin Rajwa <marcin.rajwa@linux.intel.com>
 
 #include <sof/debug/gdb/ringbuffer.h>
-#include <sof/lib/memory.h>
+#include <sof/lib/mailbox.h>
 
 #define BUFFER_OFFSET 0x120
 
-volatile struct ring * const rx = (void *) SRAM_DEBUG_BASE;
-volatile struct ring * const tx = (void *)(SRAM_DEBUG_BASE + BUFFER_OFFSET);
-volatile struct ring * const debug = (void *)(SRAM_DEBUG_BASE +
+volatile struct ring * const rx = (void *)mailbox_get_debug_base();
+volatile struct ring * const tx = (void *)(mailbox_get_debug_base() +
+					   BUFFER_OFFSET);
+volatile struct ring * const debug = (void *)(mailbox_get_debug_base() +
 					      (2 * BUFFER_OFFSET));
 
 void init_buffers(void)

